src/main/java/io/confluent/connect/jdbc/sink/DbStructure.java

```diff
@@ -172,6 +172,33 @@ boolean amendIfNecessary(
         missingFields.add(field);
       }
     }
-    return missingFields;
+
```

Comment:
Maybe return here if `missingFields.isEmpty()`?

Reply:
This list is used to create ALTER table queries. So we need to return the set here.

Reply:
Right. If there are no missing fields at this point, why go through the work of checking case-insensitively? Why not return an empty list here?